src/pages/gamer/[platform]/[username].tsx

```diff
@@ -11,27 +50,52 @@ export default function Gamers({gamerData}) {
                 </main>
             </div>
         )
-    }
-    else{
+    } else {
         return (
             <div className="container">
                 <Head>
-                    <title>Gamers</title>
+                    <title>{gamer.username}</title>
                     <link rel="icon" href="/favicon.ico"/>
                 </Head>
                 <main>
-                    <h1>{gamer.username}</h1>
-                    <h1>{gamer.platform}</h1>
+                    <GamerCard gamer={gamer}/>
+                    <div style={{"margin": "auto"}}>
+                        <Tabs activeTab={chartState.activeTab} onChange={(tabId) => setTabAndFetchData(gamer.username, gamer.platform, tabId, chartState, setChartState)} ripple>
+                            <Tab>Teammates</Tab>
+                            <Tab>Placements</Tab>
+                            <Tab>Stats</Tab>
+                            <Tab>Time</Tab>
+                        </Tabs>
+                        <section>
+                            {TabData}
+                        </section>
+                    </div>
                 </main>
             </div>
         )
     }
 }
 
+async function fetchViewData(username, platform, view) {
+    //TODO MAKE THIS WORK ON HEROKU TOO
+    let dataUrl ='https://localhost:3000/api/gamer/' + platform + '/' + username + '?view=' + view;
```

Comment:
@fjaxyu  I neeed to figure out how to declare global variables in NextJS. Referring to prcoess.env.HOSTNAME does not work here.

Reply:
Wouldn't this just be hard-coded process.env.DEV === true ? 'http://localhost:3000' : 'https://brshooter.com'?

Reply:
Nah. I figured this out. The reason why I couldn't access process.env here is that it is outside a NextJS component. I just needed to do process.env.HOSTNAME inside the NextJS component and pass it to this function and it worked great.